kubectl-rook-ceph.sh

```diff
@@ -610,6 +601,15 @@ function parse_main_flag() {
 REMAINING_ARGS=()
 parse_flags 'parse_main_flag' "$@"
 
+# Default values
```

Comment:
We have a CI test in a namespace other that rook-ceph, right? If so, I'm curious how we missed this.

Reply:
so in the CI or generally when I did testing, we are either not passing the namespace or passing both the namespace cluster and operator. That's why it is missed in CI.

The current behavior is
1. default is rook-ceph
2.  if only the cluster namespace passed then the operator will also pick the namespace as the cluster
3. we can pass both cluster and operator namespace

The only other case I can see that I don't think is important is when the user is only passing the operator namespace, in this case, cluster namespace will rook-ceph only which I think make sense if user don't pass namespace we stick with `rook-ceph`

Reply:
Agreed on that behavior, thanks for clarifying.